### PR TITLE
Refactor python build

### PR DESCRIPTION
**Pull request by blowekamp**

### CODE REVIEW

File: Wrapping/Python/CMakeLists.txt [hunk 1]
```diff
@@ -55,14 +55,14 @@ target_include_directories( ${SWIG_MODULE_SimpleITKPython_TARGET_NAME}
...
     ${CMAKE_CURRENT_SOURCE_DIR} )
 
-set_target_properties( ${SWIG_MODULE_SimpleITK_TARGET_NAME} PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
+set_target_properties( ${SWIG_MODULE_SimpleITK_TARGET_NAME} PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/SimpleITK)
 if ( MSVC )
     foreach ( CMAKE_CONFIGURATION_TYPE ${CMAKE_CONFIGURATION_TYPES} )
       string(TOUPPER ${CMAKE_CONFIGURATION_TYPE} CMAKE_CONFIGURATION_TYPE)
       set_target_properties(${SWIG_MODULE_SimpleITK_TARGET_NAME}
-        PROPERTIES LIBRARY_OUTPUT_DIRECTORY_${CMAKE_CONFIGURATION_TYPE} "${CMAKE_CURRENT_BINARY_DIR}")
+        PROPERTIES LIBRARY_OUTPUT_DIRECTORY_${CMAKE_CONFIGURATION_TYPE} "${CMAKE_CURRENT_BINARY_DIR}/SimpleITK")
```
Comment (zivy) [inline Wrapping/Python/CMakeLists.txt:63]: same as above

File: Wrapping/Python/CMakeLists.txt [hunk 2]
```diff
@@ -55,14 +55,14 @@ target_include_directories( ${SWIG_MODULE_SimpleITKPython_TARGET_NAME}
...
       set_target_properties(${SWIG_MODULE_SimpleITK_TARGET_NAME}
-        PROPERTIES RUNTIME_OUTPUT_DIRECTORY_${CMAKE_CONFIGURATION_TYPE} "${CMAKE_CURRENT_BINARY_DIR}")
+        PROPERTIES RUNTIME_OUTPUT_DIRECTORY_${CMAKE_CONFIGURATION_TYPE} "${CMAKE_CURRENT_BINARY_DIR}/SimpleITK")
```
Comment (zivy) [inline Wrapping/Python/CMakeLists.txt:65]: same as above

File: Wrapping/Python/CMakeLists.txt [hunk 3]
```diff
@@ -76,16 +76,27 @@ file( TO_NATIVE_PATH "${SimpleITK_PYTHON_PACKAGE_DIR}" SimpleITK_PYTHON_PACKAGE_
...
+
+
+set(SimpleITK_Py_Files "__init__.py" "extra.py" )
+
 if(DEFINED SKBUILD)
   # Currently this installation
   install(FILES
-      ${CMAKE_CURRENT_BINARY_DIR}/SimpleITK.py
-      ${CMAKE_CURRENT_SOURCE_DIR}/Packaging/__init__.py
+      ${CMAKE_CURRENT_BINARY_DIR}/SimpleITK/SimpleITK.py
```
Comment (zivy) [inline Wrapping/Python/CMakeLists.txt:86]: ${CMAKE_CURRENT_BINARY_DIR}/SimpleITK/SimpleITK.py 
change to:
${CMAKE_SWIG_OUTDIR}/SimpleITK.py

**Comment by blowekamp** [comment]
Thank you for the review. I have made the recommended change.

**Pull request merged**